tests/pr_test.go

```diff
@@ -17,6 +17,7 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 		Testing:       t,
 		TerraformDir:  dir,
 		Prefix:        prefix,
+		Region:        "us-south",
 		ResourceGroup: resourceGroup,
 	})
 	return options
```

Comment:
your going to want to update these tests to point to your DA code (in root directory) instead of the advanced example. You don't even need any examples in this repo since the root level code in the repo is a DA. This is an example, and this is what your going to run tests against